paramiko/kex_gex.py

```diff
@@ -87,7 +88,7 @@ def parse_next(self, ptype, m):
             return self._parse_kexdh_gex_reply(m)
         elif ptype == _MSG_KEXDH_GEX_REQUEST_OLD:
             return self._parse_kexdh_gex_request_old(m)
-        raise SSHException('KexGex asked to handle packet type %d' % ptype)
+        raise SSHException('KexGex %s asked to handle packet type %d' % self.name, ptype)
```

Comment:
This statement is missing a set of parentheses around the format arguments, it needs to be `% (self.name, ptype))`